src/redis-cli.c

```diff
@@ -3292,6 +3296,23 @@ static int isSensitiveCommand(int argc, char **argv) {
                 return 0;
             }
         }
+    } else if (argc > 4 && !strcasecmp(argv[0], "sentinel")) {
+        /* SENTINEL CONFIG SET sentinel-pass password
+         * SENTINEL CONFIG SET sentinel-user username */
+        if (!strcasecmp(argv[1], "config") && 
+            !strcasecmp(argv[2], "set") && (
+                !strcasecmp(argv[3], "sentinel-pass") ||
+                !strcasecmp(argv[3], "sentinel-user"))) {
```

Comment:
styling: with multi-line `if` we usually bring the `{` to the next line.

Reply:
Address, please check, Thanks